lib/XMLHttpRequest.js

```diff
@@ -117,6 +119,8 @@ exports.XMLHttpRequest = function () {
   this.withCredentials = false
   // "text", "arraybuffer", "blob", or "document", depending on your data needs.
   // Note, setting xhr.responseType = '' (or omitting) will default the response to "text".
+  // Actual accepted values for async are "text", "arraybuffer", or "buffer".
+  // Other values will result in "buffer" for async and "arraybuffer" (Uint8Array) for sync
```

Comment:
> The XMLHttpRequest property responseType is an enumerated string value specifying the type of data contained in the response. It also lets the author change the response type. If an empty string is set as the value of responseType, the default value of text is used.

Are you sure that invalid values are treated as `buffer` for async and `arraybuffer` for sync. I need to check but I would expect the default value to be used `text`.

Reply:
All values except 'text' and '' result in Buffer (which is an Uint8ArrayBuffer). (after a slight change).  Wording to be changed....